Dockerfile

```diff
@@ -7,24 +7,31 @@ RUN adduser --system \
 	--group \
 	deis
 
-COPY requirements.txt /app/requirements.txt
+# Install permanent dependencies
 
-RUN buildDeps='gcc git libffi-dev libpq-dev python3-dev'; \
-	apt-get update && \
+RUN apt-get update && \
 	apt-get install -y --no-install-recommends \
-        $buildDeps \
-        libpq5 \
-        python3 \
-        sudo && \
-    ln -s /usr/bin/python3 /usr/bin/python && \
+				libpq5 \
+				python3 \
+				sudo && \
+		ln -s /usr/bin/python3 /usr/bin/python && \
 	curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.2 && \
 	mkdir -p /configs && chown -R deis:deis /configs && \
+	apt-get clean && \
```

Comment:
Yes, because you have to cleanup in the same layer you add the files. Deleting files in a different would make the image larger.